inst/qml/InformedMultinomialTestBayesian.qml

```diff
@@ -0,0 +1,277 @@
+		{
+			name:				"allVariablesList"
+		}
+
+		AssignedVariablesList
+		{
+			id:					factors
+			name:				"factor"
+			title:				qsTr("Factor")
+			singleVariable:		true
+			suggestedColumns:	["ordinal", "nominal"]
+		}
+
+		AssignedVariablesList
+		{
+			name:				"count"
+			title:				qsTr("Counts")
+			singleVariable:		true
+			suggestedColumns:	["scale", "ordinal"]
+		}
+	}
+
+	ColumnLayout
+	{
+
+		RadioButtonGroup
+		{
+			title:		qsTr("Bayes Factor Comparison")
+			name:		"bfComparison"
+
+			RadioButton
+			{
+				value:		"encompassing"
+				label:		qsTr("Encompassing")
+				checked:	true
+			}
+
+			RadioButton
+			{
+				value:		"null"
+				label:		qsTr("Null")
+			}
+
+			RadioButton
+			{
+				name:				"vs"
+				label:				qsTr("vs.")
+				childrenOnSameRow:	true
+
+				DropDown
+				{
+					source:				[ models ]
+					name:				"bfVsHypothesis"
+					id:					bfTypevsName
+					indexDefaultValue:	0
+				}
+			}
+		}
+
+		BayesFactorType { }
+
+	}
+
+	ColumnLayout
+	{
+
+		CheckBox
+		{
+			name:	"descriptivesTable"
+			label:	qsTr("Descriptives")
+
+			CheckBox
+			{
+				name:				"descriptivesTableCi"
+				label:				qsTr("Credible interval")
+				childrenOnSameRow:	true
+
+				CIField
+				{
+					name:	"descriptivesTableCiCoverage"
+				}
+			}
+		}
+
+		Group
+		{
+			title	: qsTr("Plots")
+
+			CheckBox
+			{
+				name:		"descriptivesPlot"
+				id:			descriptivesPlot
+				label:		qsTr("Descriptives")
+			}
+
+			CheckBox
+			{
+				name:		"posteriorPlot"
+				id:			posteriorPlot
+				label:		qsTr("Posterior")
+			}
+
+			CIField
+			{
+				enabled:	descriptivesPlot.checked || posteriorPlot.checked
+				name:		"descriptivesAndPosteriorPlotCiCoverage"
+				label:		qsTr("Credible interval")
+			}
+		}
+
+		RadioButtonGroup
+		{
+			name	: "display"
+			title	: qsTr("Display")
+
+			RadioButton
+			{
+				value:		"counts"
+				label:		qsTr("Counts")
+				checked:	true
+			}
+
+			RadioButton
+			{
+				value:		"proportions"
+				label:		qsTr("Proportions")
+			}
+		}
+	}
+
+	ExpanderButton
+	{
+		title	: qsTr("Prior")
+
+		Text
+		{
+			visible: factors.count == 0
+			text: qsTr("No factor specified")
+		}
+
+		Chi2TestTableView
+		{
+			name			: "priorCounts"
+
+			source			: "factor"
+			visible			: factors.count > 0
+
+			showAddButton		: false
+			showDeleteButton	: false
+			colHeader			: qsTr("Counts")
+			cornerText			: qsTr("Level #")
+			itemType			: JASP.Integer
+		}
+	}
+
+
+	Section
+	{
+		
+		title:		qsTr("Order Restricted Hypotheses")
+		columns:	2
+		expanded:	true
+		
+		Text
+		{
+			Layout.columnSpan:	1
+			text:				qsTr("Enter each restriction of one hypothesis on a new line,\ne.g., Low <= Mid <= High\nwhere 'Low'/'Mid'/'High' are the factor level names.\nClick on the 'plus' icon to add more hypotheses. \nClick the information icon for more examples.")
+		}
+
+		HelpButton
+		{
+			toolTip:			qsTr("Click to learn more about the syntax for order restrictions.")
+			helpPage:			"goric/restriktorSyntax"
```

Comment:
this doesn't work for me, did it work for you? Could also be something Ubuntu related.

Reply:
> this doesn't work for me, did it work for you? Could also be something Ubuntu related.

What version of JASP do you use? it may be fixed by https://github.com/jasp-stats/jasp-desktop/pull/5049

Reply:
it doesn't work when I build on Linux either -- I can check when I switch back to Windows in the afternoon

Reply:
hmm, I just tried Windows Nightly + development module and it didn't work